js/widgets/fixedToolbar.js

```diff
@@ -97,34 +94,36 @@ return $.widget( "mobile.toolbar", $.mobile.toolbar, {
 	},
 
 	_addTransitionClass: function() {
```

Comment:
If transition is only used by ```tapToggle``` this code should be moved to taptoggle.js.
Also, the transition class should only be added if ```tapToggle``` is ```true```. Currently (1.4.5) the transition class is also added when ```tapToggle``` is set to ```false```.